_rules/links-identical-name-equivalent-purpose-b20e66.md

```diff
@@ -24,25 +25,31 @@ htmlHintIgnore:
 
 ## Applicability
 
-This rule applies to any set of any two or more HTML or SVG elements that have the [semantic role](#semantic-role) of `link`, or a role that inherits from the `link` role, are [included in the accessibility tree](#included-in-the-accessibility-tree), and that have [matching](#matching-characters) [accessible names][accessible name] that do not only consist of [whitespace](#whitespace).
+This rule applies to any set of any two or more HTML or SVG elements which
 
-**Note:** The test target for this rule is the full set of link elements that share the same [matching](#matching-characters) [accessible name][].
+- have the [semantic role][] of `link`, or a role that inherits from the `link` role; and
+- are in [documents][document] of the same [web page][]; and
```

Comment:
I think it is linked to passed example 13. But I also don't understand it.

Reply:
I think I understand now. Without this clause the links can be any two links on the internet. This makes sure they are on the same web page. But the "are in documents" is really confusing and not necessary.

It would be enough to simply say "are on the same web page (HTML)"

Reply:
> I think I understand now. Without this clause the links can be any two links on the internet. This makes sure they are on the same web page. But the "are in documents" is really confusing and not necessary.
> 
> It would be enough to simply say "are on the same web page (HTML)"

The definition of "web page (HTML)" is a set of documents. Thus, links are technically not on the same web page, but are merely part of documents which appear on the same web page.

At the same time, I do agree that this is maybe over-technicalling the rule and that abusively writing "elements on the same web page (HTML)" instead of the correct "elements in documents that are part of the same web page (HTML)" is probably OK. I can also add a note on the definition to make that clear.

Reply:
I simplified the formulation here and added a note in the definition to justify this simplification. Please check that this is OK.